### PR TITLE
Define redis url for publisher

### PR DESCRIPTION
**Pull request by mgrassotti**
Define environment constants for publisher, in order to configure the
govuk_sidekiq gem properly.

### CODE REVIEW

**Comment by mgrassotti** [comment]
This is related to https://github.com/alphagov/publisher/pull/483

**Comment by alexmuller** [comment]
I think this is similar to the comment I added on #4582.

File: modules/govuk/manifests/apps/publisher.pp [hunk 1]
```diff
@@ -106,5 +108,11 @@
       app     => 'publisher',
       varname => 'SECRET_KEY_BASE',
       value   => $secret_key_base;
+    "${title}-REDIS_HOST":
+      varname => 'REDIS_HOST',
```
Comment (boffbowsh) [inline modules/govuk/manifests/apps/publisher.pp]: This is missing the `app` variable, which is why the tests are failing

Comment (mgrassotti) [inline modules/govuk/manifests/apps/publisher.pp]: 👍 thanks

**Comment by boffbowsh** [comment]
LGTM merge on Thursdasy 👍

**Pull request merged**